test/components/idea_controls_test.js

```diff
@@ -4,6 +4,7 @@ import sinon from "sinon"
 
 import IdeaControls from "../../web/static/js/components/idea_controls"
 import VoteCounter from "../../web/static/js/components/vote_counter"
+import voteMax from "../../web/static/js/configs/retro_configs"
```

Comment:
this was confusing for me, i think due to the naming, because `voteMax` is the default export of a file that I would think exported an object. rather than having `retro_configs` have a default export of `voteMax`, it could just export the constant, and then here, we could just grab that via:
```es6
import { voteMax } from "../../web/static/js/configs/retro_configs"
```